docs/currency.md

```diff
@@ -89,26 +91,19 @@ Remote currencies usually have their respective bridge identifiers attached in t
 
 You may find example in XCM and XCVM documentation.
 
-## Native currencies and gas fees
-
-Users pay fees in the runtime's native currency for the execution, allowing the runtime to earn and prevents spam and DDoS attacks.
+## Currencies and gas fees
 
-A runtime configuration allows mapping amount of execution which can be bought for amount of native currency.
-
-This parachain uses pre-calculated weights to estimate the hardware costs of execution. An amount of weight a native currency can buy is a non-linear dynamic function from currency amount to weight. As the usage of this chain increases towards maximum capacity, the price of a unit of weight increases as well. 
+See [BYOG](./byog.md) for details.
 
 Cross-chain interactions over bridges may involve metered networks which allow setting limits on execution price and calculating the possible number of execution resources in the runtime. For example, Ethereum.
 
-
-Within the protocols built on top of the parachain, there are "native" protocol currencies that are used for governance and revenue distribution, but _cannot_ be used to pay for execution.
+Within the protocols built on top of the parachain, there are "native" protocol currencies that are used for governance and revenue distribution, these may be use to pay gas fees of configured that way.
 
 ### Bridged gas fee (`bring your own gas`)
 
-Users who want to have assets on our chain can either be minted with some native currency governance or be allowed to pay for cross-chain transactions in some non-native currency.
-
-Currently, non-native currencies are mapped to native with some configurable ratio which in turn is mapped to a weight.
+See [BYOG](./byog.md) for details.
 
-Transactions issued by users are likely to swap some remote currencies into native on transfer, to enact the execution of local transactions. 
+Users who want to have assets on our chain can either be minted with some native currency governance or be allowed to pay for cross-chain transactions in some non-native currency.
```

Comment:
```suggestion
Users who want to have assets on our chain can either mint with some native currency governance or be allowed to pay for cross-chain transactions in some non-native currency.
```
Nit expression

Reply:
so in this case they do not not mint. but some sudo/protocol/admin mints to them (so user are passibe and minted to them. not sure how make it clear. but they do not mint.

Reply:
hm, not that true too. he may be not requested, but that happend by some protocol. request can be one of these, or airdrop, etc. so `on chain protocol mints`

Reply:
> hm, not that true too. he may be not requested, but that happend by some protocol. request can be one of these, or airdrop, etc. so `on chain protocol mints`

Reading this, my understanding is: 
"Users can take certain actions that trigger some on-chain protocol to mint assets to the user for native currency."

If users don't mint themself and don't request some entity to mint, I imagine users take some action for a protocol to react by minting to the user, is that correct?

Reply:
> Users can take certain actions that trigger some on-chain protocol to mint assets to the user for native currency

They can participate in offchain crowloans or other marketing, and they may be rewarded with amounts by us.

Reply:
> If users don't mint themself and don't request some entity to mint, I imagine users take some action for a protocol to react by minting to the user, is that correct?
don't request some entity to mint = take some action for a protocol  = bring some proof so some on chain entity that that request is allowed to execute and it will mint. so it is same. you prove to protocol to do some action, and that action can mint you or else account PICA. can be airdrop, LBP, crowdloans, XCVM swaps, sudo asset calls.

Reply:
```suggestion
Users who want to have assets on our chain can either be issued native currency by some on-chain protocol or be allowed to pay for cross-chain transactions in some non-native currency.
```
Is this accurate?

Reply:
yes.